scripts/scr_enemy_ai_e/scr_enemy_ai_e.gml

```diff
@@ -1,801 +1,943 @@
+                    if (obj_controller.recruiting = 3) then recruit_chance = floor(random(150)) + 1;
+                    if (obj_controller.recruiting = 4) then recruit_chance = floor(random(125)) + 1;
+                    if (obj_controller.recruiting = 5) then recruit_chance = floor(random(100)) + 1;
+
+                    // 135; recruiting
+                    // corr isn't really relevant as corruption in marines doesn't matter
+                    // by default it takes 72 turns (6 years) to train
+
+
+                    if (p_type[run] = "Hive") and(recruit_chance <= 40) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Temperate") and(recruit_chance <= 20) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Feudal") and(recruit_chance <= 20) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Forge") and(recruit_chance <= 15) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Shrine") and(recruit_chance <= 15) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Desert") and(recruit_chance <= 15) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Ice") and(recruit_chance <= 15) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Agri") and(recruit_chance <= 10) {
+                        aspirant = 1;
+                    } // there's no reason to use agri's
+                    if (p_type[run] = "Death") and(recruit_chance <= 10) {
+                        aspirant = 1;
+                    }
+                    if (p_type[run] = "Lava") and(recruit_chance <= 7) {
+                        aspirant = 1;
+                    }
+
+                    // if a planet type has less than half it's max pop, you get 20% less spacey marines
+                    if (_planet_population <= halfpop) {
+                        recruit_chance += 1.2;
+                        scr_alert("red", "owner", "The populations you attain aspirants from are less populant than required, chances of recruiting aspirants is 20% lower", 0, 0);
+                    }
+
+                    // This is the area has trial types that don't care about planet type 
+                    // xp is given in a latter if loop
+
+                    if (obj_controller.recruit_trial = "Blood Duel") { // blood duel is most numerous, but not great with gene seed
+                        months_to_neo -= choose(24, 24, 36, 36, 36, 48);
+                        corr += choose(10, 15, 20);
+                        recruit_chance -= choose(0.7, 0.7, 0.8, 0.8, 0, 8, 0.9);
+
+                        if (obj_controller.recruiting > 0) {
```

Comment:
actual fix, everything is just beautification with https://beautifier.io/, removal of some commented out code, and tiny changes, like `var a; a = 1` to `var a = 1`